lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -1078,8 +1078,7 @@ class RefreshSystemLockLayer : public BaseCmd
 
     void _notifySystemLockIsRefreshed()
     {
-        auto dstCallbacks = UsdUfe::getUICallbacks(TfToken("onRefreshSystemLock"));
-        if (dstCallbacks.size() == 0)
+        if (!UsdUfe::isUICallbackRegistered(TfToken("onRefreshSystemLock")))
```

Comment:
New trigger callback function and removed the get. I think this is safer than allowing code to access all the callbacks.